libdevice/cmake/modules/SYCLLibdevice.cmake

```diff
@@ -444,7 +451,7 @@ foreach(dtype IN ITEMS bf16 fp32 fp64)
   endforeach()
 endforeach()
 
-# Add device fallback imf libraries for the CUDA target.
+# Add device fallback imf libraries for the CUDA and AMD targets.
```

Comment:
We seem to be mixing terminology here, which is understandable and it happens often, but `CUDA` and `AMD` stand out to me here. Either NVPTX/AMD or CUDA/HIP?